routes/progress.js

```diff
@@ -0,0 +1,98 @@
+require("dotenv").config();
+const express = require("express");
+const router = express.Router();
+let progressDB = require("../progressDatabase.js");
+const jwt = require("jsonwebtoken");
+
+router.get("/", authenticateToken, (req, res) => {
+  let userId = 1; //This should not be hard coded. Should get userId from JWT in future
+
+  let select = "SELECT language,category,progress FROM progress WHERE userId=?";
+  let params = [userId];
+  const rowData = [];
+  progressDB.all(select, params, (error, rows) => {
+    if (error) {
+      console.log(error);
+      res.status(500).send();
+    }
+    rows.forEach((row) => {
+      rowData.push(row);
+    });
+    res.json(rowData).send();
+  });
+});
+
+router.post("/update", async (req, res) => {
```

Comment:
```suggestion
router.post("/update", (req, res) => {
```
Does async have any function here?

Reply:
oh I missed this - however i think we should try it and not just make changes from github :)

Reply:
Agreed